commit 1ace82c7f9afee232c7e39f533fb40d636941090
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Sat Aug 27 16:28:50 2022 +0200

    media: atomisp: Remove atomisp_file_fops and atomisp_file_ioctl_ops
    
    After the file-injection support removal, atomisp_video_pipe->type never
    is V4L2_BUF_TYPE_VIDEO_OUTPUT anymore, so the V4L2_BUF_TYPE_VIDEO_OUTPUT
    support path in atomisp_video_init() is never hit and this path is
    the only user of atomisp_file_fops and atomisp_file_ioctl_ops.
    
    Remove atomisp_file_fops and atomisp_file_ioctl_ops and all of
    the functions which are only referenced by these ops structs.
    
    Reviewed-by: Andy Shevchenko <andriy.shevchenko@intel.com>
    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@kernel.org>

diff --git a/drivers/staging/media/atomisp/pci/atomisp_cmd.h b/drivers/staging/media/atomisp/pci/atomisp_cmd.h
index ebc729468f87..ed1ad5389104 100644
--- a/drivers/staging/media/atomisp/pci/atomisp_cmd.h
+++ b/drivers/staging/media/atomisp/pci/atomisp_cmd.h
@@ -269,7 +269,6 @@ int atomisp_try_fmt(struct video_device *vdev, struct v4l2_pix_format *f,
 		    bool *res_overflow);
 
 int atomisp_set_fmt(struct video_device *vdev, struct v4l2_format *f);
-int atomisp_set_fmt_file(struct video_device *vdev, struct v4l2_format *f);
 
 int atomisp_set_shading_table(struct atomisp_sub_device *asd,
 			      struct atomisp_shading_table *shading_table);